### ISSUE TITLE
Compatibility v13 (and v12 too)

### ISSUE THREAD

**Issue by DavidBruchmann** [FriendsOfTYPO3/extension_builder#767]
**Is your feature request related to a problem? Please describe.**
The current development version runs at best in TYPO3 v12 only.
As v13 LTS will be available at some time too, it would be great if the extension-builder would be available then too.

**Describe the solution you'd like**
An extension-builder that is running in v13.

**Additional context**
Some issues are still related to v12 too.
So in a first step all things will be changed that relate to v12 too, and those issues can be used for both versions.
Later it has to be seen if there are issues only applying to v13.

**Comment by DavidBruchmann** [comment]
I created a new branch `main-dev`, which will be used for the explained purpose.

**Comment by DavidBruchmann** [comment]
yeah, just keep the workflow in mind, in the beginning it's still about v12 too.

**Comment by DavidBruchmann** [comment]
see #769

**Comment by PKuhlmay** [comment]
I close this, as it is already released for v13 today.
